src/interactions/interaction.ts

```diff
@@ -24,17 +25,22 @@ export class Interaction {
    * @returns {Interaction} The calling Interaction.
    */
   public attachTo(component: Component) {
-    if (this._componentAttachedTo) {
-      this.detachFrom(this._componentAttachedTo);
-    }
-
+    this._detach();
     this._componentAttachedTo = component;
-    component.onAnchor(this._anchorCallback);
-
+    this._attach();
     return this;
   }
 
   /**
+   * Attaches to existing _componentAttachedTo if enabled.
```

Comment:
Oh I just realized that this is documentation on a private function...  Never mind, I think? this is fine.  Not sure, but I think this is ok.